src/renderer/api/types.ts

```diff
@@ -766,6 +766,18 @@ export type RatingQuery = {
 
 export type SetRatingArgs = { query: RatingQuery; serverId?: string } & BaseEndpointArgs;
 
+// Sharing
+export type ShareItemResponse = { id: string } | undefined;
+
+export type ShareItemBody = {
+    description: string;
+    downloadable: boolean;
+    resourceIds: string;
```

Comment:
Similar, should probably also have expiry here as well (optional, either date or unix timestamp)